bin/CoreData/Shaders/GLSL/Skybox.glsl

```diff
@@ -6,7 +6,13 @@ varying vec3 vTexCoord;
 
 void VS()
 {
+#ifdef IGNORENODETRANSFORM
+    mat4 m = transpose(cViewInv); 
+    m[0] = vec4(1,0,0,0); m[1] = vec4(0,1,0,0); m[2] = vec4(0,0,1,0);

```

Comment:
Use 1.0 instead 1 to compatibility with old OpenGL https://github.com/1vanK/Urho3DOutline/pull/1
Also spaces afer ,